service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -137,7 +148,8 @@ private SiteFeature siteFeatureFromRecord(
             sitesRecord.value4(),
             sitesRecord.value5(),
             sitesRecord.value6(),
-            sitesRecord.value7());
+            sitesRecord.value7(),
+            sitesRecord.value10());
```

Comment:
❓ Just for consistency should we switch around the order in the record so that the Date is value8?

Reply:
Done